tools/lsp/main.rs

```diff
@@ -3,6 +3,8 @@
 
 #![cfg(not(target_arch = "wasm32"))]
 
+#![allow(clippy::await_holding_refcell_ref)]
```

Comment:
Maybe could get a comment. Is it not possible to put that on the particular refcell which are used within await?